tests/ws/driver_test.go

```diff
@@ -0,0 +1,453 @@
+package ws
+
+import (
+	"context"
+	"database/sql"
+	"os"
+	"testing"
+	"time"
+
+	_ "github.com/libsql/libsql-client-go/libsql"
+)
+
+func Test_QueryExec(t *testing.T) {
```

Comment:
We name tests without _ in our projects. So TestQueryExec in this case but this comment applies to all tests.